### PR TITLE
main/clamav: security upgrade to 0.100.2

### PR DESCRIPTION
**Pull request by J0WI**
https://github.com/Cisco-Talos/clamav-devel/blob/dev/0.100/NEWS.md#01002
Please backport this to stable releases.

### CODE REVIEW

File: main/clamav/APKBUILD [hunk 1]
```diff
@@ -53,6 +53,11 @@ builddir="$srcdir/$pkgname-$pkgver"
 #     - CVE-2017-16932
 #     - CVE-2018-0360
 #     - CVE-2018-0361
+#   0.100.2-r0:
+#     - CVE-2018-15378
+#     - CVE-2018-14680
+#     - CVE-2018-14681
+#     - CVE-2018-14682
```
Comment (andypost) [inline main/clamav/APKBUILD]: Makes sense to mention CVEs in commit message

Comment (J0WI) [reply main/clamav/APKBUILD]: Done

**Comment by alexxkn** [comment]
Hello, @andypost :)
Could we somehow speed up the process of MR acceptance?

**Comment by J0WI** [comment]
https://git.alpinelinux.org/cgit/aports/commit/?id=5412962cc2f34d4bb2f2996918e1384eda223946

**Pull request closed without merging**